src/theme/css/chrome.css

```diff
@@ -262,6 +262,26 @@ pre > .buttons button {
     transition-property: color,border-color,background-color;
     color: var(--icons);
 }
+
+pre > .buttons button.clip-button {
+    padding: 2px 4px 0px 6px;
+}
+pre > .buttons button.clip-button::before {
+    /* clipboard <https://github.com/rust-lang/crates.io/commits/main/public/assets/copy.svg> */
```

Comment:
It looks like this was taken from [octicons](https://github.com/primer/octicons) (particularly, somewhere around the [2.0](https://github.com/primer/octicons/blob/v2.0.0/svg/clippy.svg) release). Can you update this to include the provenance and license from that version?

Reply:
Done!